modules/ROOT/pages/get-started.adoc

```diff
@@ -35,17 +35,21 @@ Higher level permissions override the *Governance Viewer* permission. If a user
  
 If you're an administrator and have an opt-in banner in the API Governance console, opt in to get additional functionality for your business groups (organizations). See the xref:release-notes::api-governance/api-governance-release-notes.adoc##oct-17-2023[API Governance Release Notes entry for October, 17 2023].
 
-This documentation is written for the full product functionality that is available after the opt-in.
+This documentation covers the full product functionality that is available after the opt-in.
 
 == Govern Your APIs
 
 Govern your APIs using the following steps:
 
-. xref:add-tags.adoc[Identify] the APIs that you want to govern.
+. xref:add-tags.adoc[Identify] the APIs in Exchange that you want to govern by using tags and categories.
```

Comment:
```suggestion
. xref:add-tags.adoc[Identify] the APIs in Exchange to govern by using tags and categories.
```

Reply:
It might just be me, but it seems that the shortened version implies that using the tags and categories will automatically govern them right then vs. requiring some additional action. I also still like a tiny little friendliness over conciseness, so I'll leave this one as-is for now. I've gotten rid of most of my "want to", but in the Getting Started, it's still trying to ease them in. Maybe later I'll get more strict about it. :)

Reply:
I've changed this to the following, which is hopefully makes it more clear. 

. xref:add-tags.adoc[Identify] APIs to filter for governance by using tags and categories in Exchange.